CHANGELOG.md

```diff
@@ -1,5 +1,54 @@
 # Changelog
 
+## 0.42.0 / 2023-12-07
+
+- [FEATURE] Initial Tempo Plugin (#1561)
+- [FEATURE] Add authentication system (#1472) (#1473) (#1476) (#1478) (#1484) (#1547) (#1555) (#1557) (#1591) (#1602) (#1612) (#1615)
+- [FEATURE] Add authorization system based on rbac (#1570) (#1511) (#1613) (#1601) (#1605) (#1626)
+- [FEATURE] Adding explore page (#1477) (#1592) (#1610)
+- [FEATURE] Enabling data provisioning (#1486)
+- [FEATURE] Add sorting capability for ListVariable (#1502)
+- [FEATURE] Add Secret & GlobalSecret list view (#1501)
+- [FEATURE] Display the server config (#1496)
+- [ENHANCEMENT] Prevent using dashboard datasources before they're saved (#1603)
+- [ENHANCEMENT] add a default encryption key (#1599)
+- [ENHANCEMENT] Use a context for the config to share it across the whole app (#1604)
+- [ENHANCEMENT] Implement the `healthcheck` method for Prometheus datasources (#1595)
+- [ENHANCEMENT] Local datasources CRUD (#1587)
```

Comment:
This one is more a feature actually - my bad on the wrong flagging here. Could you fix it in the changelog?